runtime/lua/vim/treesitter/query.lua

```diff
@@ -284,7 +285,15 @@ M.get = memoize('concat-2', function(lang, query_name)
   end
 
   return M.parse(lang, query_string)
-end)
+end, false)
+
+api.nvim_create_autocmd('OptionSet', {
+  pattern = { 'runtimepath' },
+  group = api.nvim_create_augroup('ts_query_cache_reset', { clear = true }),
```

Comment:
For namespaces let's choose a consistent convention, I suggest `nvim.foo.bar` instead of undescores/slashes/whatever else we're doing.